libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -954,7 +952,8 @@ void ContractCompiler::handleCatch(vector<ASTPointer<TryCatchClause>> const& _ca
 				revert(0, returndatasize())
 			})");
 		else
-			m_context.appendRevert();
+			// Since both returndata and revert are >=byzantium, this should be unreachable.
+			solAssert(false, "");
```

Comment:
Does this make sense?

Reply:
Can it be triggered?

Reply:
Not by any of our tests, and I couldn't manually.